lib/page-object/platforms/watir_webdriver/page_object.rb

```diff
@@ -918,6 +918,23 @@ def elements_for(tag, identifier)
         end
 
         #
+        # platform method to return a specified class wrapping an element
+        # See PageObject::Accessors#page_section
+        #
+        def page_section_for(section_class, identifier)
+          find_watir_element("element(identifier)", section_class, identifier, 'element')
```

Comment:
the tag of the element is less important for this feature since the relative value of the element isn't for anything but for the fact that it contains other elements. I considered trying to support adding the tag_name to the identifiers, but I haven't bitten that off. Let me know if you see enough need for it.